content/en/docs/setup/scratch.md

```diff
@@ -766,10 +767,13 @@ Template for controller manager pod:
 Flags to consider using with controller manager:
 
  - `--cluster-cidr=`, the CIDR range for pods in cluster.
+ - `--allocate-node-cidrs=true`, allow CIDRs allocated for Node to make CNI plugins work properly.
```

Comment:
Can we just add this context to the option on the next line? This also is not enough for the [CNI use-case](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#cni) if do not include `--cluster-cidr=<cidr>`